### PR TITLE
Overview of developing serverless applications in C#

### LINKED ISSUE #5313: [Cloud Computing] Overview of developing serverless applications in c#
#### Proposed title of article
Overview of developing serverless applications in c#

#### Proposed article introduction
Serverless computing has with no doubt revolutionalized the world of computing. Developers are finding fun using it since it has made their development efficient and more productive.
Before serverless, developers had to deal with resources that support their code. Since the launch of serverless, developers worry no more about the operating system and hardware architecture since serverless handles the server infrastructure. This gives developers more time to focus on writing good and quality code. 
This article will draw more attention to building a serverless function using c# and later be deployed on cloud providers such as AWS and Azure.

#### Key takeaways 
At the end of the article, the reader will understand:
- the usage of c# with Azure Functions and AWS Lambda
- the steps of writing c# code for AWS Lambda with Visual Studio
- how to run and publish to AWS Lambda

#### Article quality
The proposed article will focus in detail on both serverless platforms, Azure Functions, and AWS Lambda, unlike the existing articles which focus on the specific platforms.

#### References
N/A

### PR DESCRIPTION
**Pull request by ephraa**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #5313

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Good afternoon and thank you for submitting your article for publishing.
Your article has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. :rocket:
Thank you.

**Comment by lalith1403** [comment]
plagiarism check done ✔️

**Comment by Bayler** [comment]
@ephraa Please fix your links.

`/overview-of-developing-serverless-applications=in-c/aws-window.PNG`

Remove the `=` sign and make sure the PNG is in small letters. When making the changes, don't forget to fix the links inside your article too.

**Comment by ephraa** [comment]
> @ephraa Please fix your links.
> 
> `/overview-of-developing-serverless-applications=in-c/aws-window.PNG`
> 
> Remove the `=` sign and make sure the PNG is in small letters. When making the changes, don't forget to fix the links inside your article too.

Hi @Bayler, the minor edits have been made. Kindly proceed with the review process.
Thank you.

**Comment by Bayler** [comment]
> > @ephraa Please fix your links.
> > `/overview-of-developing-serverless-applications=in-c/aws-window.PNG`
> > Remove the `=` sign and make sure the PNG is in small letters. When making the changes, don't forget to fix the links inside your article too.
> 
> Hi @Bayler, the minor edits have been made. Kindly proceed with the review process. Thank you.

@ephraa Your file names still have .PNG

**Comment by ephraa** [comment]
> > > @ephraa Please fix your links.
> > > `/overview-of-developing-serverless-applications=in-c/aws-window.PNG`
> > > Remove the `=` sign and make sure the PNG is in small letters. When making the changes, don't forget to fix the links inside your article too.
> > 
> > 
> > Hi @Bayler, the minor edits have been made. Kindly proceed with the review process. Thank you.
> 
> @ephraa Your file names still have .PNG

@Bayler I have fixed the links and renamed the file names. Thank you.

File: content/articles/overview-of-developing-serverless-applications-in-c/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+To achieve that, we will browse to the VS Code extension manager located at the left side of the window and search for the [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-azureextensionpack) and install it, as shown in the figure below:
+
+![Extension pack](/engineering-education/overview-of-developing-serverless-applications-in-c/azure-extensions1.png)
+
+Next, will be to sign in to Azure by pressing the keys `CTRL+Shift+P` on the keyboard to launch a Command Palette in the VS Code editor.
+
+Then, a developer will enter the `Azure: Sign In` in the text field displayed on top of VS Code editor.
+
+The developer will click on the results that appear after typing on the text field. VS Code editor will generate a code and display the following steps to be followed in order to complete the set-up.
```
Comment (Bayler) [inline content/articles/overview-of-developing-serverless-applications-in-c/index.md:54]: ...the following steps? Or just `steps`? Following steps seems like you want to list the steps to be followed.

File: content/articles/overview-of-developing-serverless-applications-in-c/index.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+#### Using VS Code
+It is a powerful and lightweight code editor that supports different operating systems such as Windows, Linux, and Mac OS. Developers can download either of the versions depending on their platforms.
+
+Azure Tools for VS Code are required to allow the developers to use the commands that interact with Azure functions in the VS Code editor.
+
+To achieve that, we will browse to the VS Code extension manager located at the left side of the window and search for the [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-azureextensionpack) and install it, as shown in the figure below:
+
+![Extension pack](/engineering-education/overview-of-developing-serverless-applications-in-c/azure-extensions1.png)
+
+Next, will be to sign in to Azure by pressing the keys `CTRL+Shift+P` on the keyboard to launch a Command Palette in the VS Code editor.
```
Comment (Bayler) [inline content/articles/overview-of-developing-serverless-applications-in-c/index.md:50]: Please fix this sentence. It's not clear

**Comment by ephraa** [comment]
Hi @Bayler, the minor changes suggested has been fixed. Thank you.

**Comment by hectorkambow** [comment]
Jumping on this one

**Pull request merged**